db/schema.rb

```diff
@@ -10,161 +10,167 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180316120551) do
-
+ActiveRecord::Schema.define(version: 20_180_326_084_545) do
```

Comment:
`db/*` files should be in rubocop exceptions, because next running of `rails db:migrate` will back it again to rails style

Reply:
add `.rubocop.yml` file with the next rules:
```yaml
AllCops:
  Exclude:
    - 'db/**/*'
  TargetRubyVersion: 2.5

Rails:
  Enabled: true
```